### PR TITLE
Add function name to spec-transformed arrow functions

### PR DESCRIPTION
**Pull request by Kovensky**
| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | yes
| Major: Breaking Change?  | no
| Minor: New Feature?      | yes
| Deprecations?            | no
| Spec Compliancy?         | yes
| Tests Added/Pass?        | yes
| Fixed Tickets            | ?
| License                  | MIT
| Doc PR                   | 
| Dependency Changes       | 

While it may appear that this should be done by es2015-function-name,
another way to think about it is that es2015-function-name implements
the naming that javascript engines are supposed to do; and javascript
engines cannot name function expressions that are the object of a
member expression.

### CODE REVIEW

File: packages/babel-plugin-transform-es2015-arrow-functions/src/index.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
-export default function ({ types: t }) {
+// @flow
+
+import nameFunction from "babel-helper-function-name";
```
Comment (loganfsmyth) [inline packages/babel-plugin-transform-es2015-arrow-functions/src/index.js]: Looks like we need this in `package.json`?

**Comment by loganfsmyth** [comment]
LGTM otherwise.

File: packages/babel-plugin-transform-es2015-arrow-functions/src/index.js (hunk 1 above)
Comment (Kovensky) [inline packages/babel-plugin-transform-es2015-arrow-functions/src/index.js]: Hm, probably.

Would be nice if lerna could take care of it 😄

**Comment by xtuc** [comment]
Could you please update the example in https://github.com/babel/babel/blob/ef6f58ad22cd1399054a781294cd1bfb06ef2a97/packages/babel-plugin-transform-es2015-arrow-functions/README.md

Otherwise looks good to me.

**Comment by hzoo** [comment]
We don't have an example for spec mode, did you mean we should add one @xtuc? Do you want to add it and squash merge?

**Comment by 792022280268** [comment]
no очень устал извините, если, что можете, Вы, а я спать,мне плохо,извините02.05.2017, 00:56, "Henry Zhu" <notifications@github.com>:We don't have an example for spec mode, did you mean we should add one @xtuc? Do you want to add it and squash merge?

**Comment by xtuc** [comment]
I would like to add an example for spec mode but I will probably do a separate PR.

**Pull request merged**